manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1829,21 +1841,50 @@ def _create_axis(
         axis.shift(-axis.number_to_point(self._origin_shift([axis.x_min, axis.x_max])))
         return axis
 
-    def coords_to_point(self, *coords: Sequence[float]) -> np.ndarray:
+    def coords_to_point(
+        self, *coords: Sequence[float] | Sequence[Sequence[float]] | np.ndarray
+    ) -> np.ndarray:
         """Accepts coordinates from the axes and returns a point with respect to the scene.
 
         Parameters
         ----------
         coords
             The coordinates. Each coord is passed as a separate argument: ``ax.coords_to_point(1, 2, 3)``.
 
+            Also accepts a list of coordinates
+
+            ``ax.coords_to_point( [x_0, x_1, ...], [y_0, y_1, ...], ... )``
+
+            ``ax.coords_to_point( [[x_0, y_0, z_0], [x_1, y_1, z_1]] )``
+
         Returns
         -------
         np.ndarray
             A point with respect to the scene's coordinate system.
+            The shape of the array will be similar to the shape of the input.
 
         Examples
         --------
+
+        .. code-block:: python
+
+            from manim import Axes
+            import numpy as np
+
+            ax = Axes()
+            np.around(ax.coords_to_point(1, 0, 0), 2)
+            # array([0.86, 0.  , 0.  ])
+            np.around(ax.coords_to_point([[0, 1], [1, 1], [1, 0]]), 2)
+            # array([[0.  , 0.75, 0.  ],
+            #        [0.86, 0.75, 0.  ],
+            #        [0.86, 0.  , 0.  ]])
+            np.around(
+                ax.coords_to_point([0, 1, 1], [1, 1, 0]), 2
+            )  # Transposed version of the above
+            # array([[0.  , 0.86, 0.86],
+            #        [0.75, 0.75, 0.  ],
+            #        [0.  , 0.  , 0.  ]])
```

Comment:
Is there a particular reason why you decided against writing this as a doctest?

```suggestion
        .. code-block:: python

            >>> from manim import Axes
            >>> import numpy as np
            >>> ax = Axes()
            >>> np.around(ax.coords_to_point(1, 0, 0), 2)
            array([0.86, 0.  , 0.  ])
            >>> np.around(ax.coords_to_point([[0, 1], [1, 1], [1, 0]]), 2)
            array([[0.  , 0.75, 0.  ],
                   [0.86, 0.75, 0.  ],
                   [0.86, 0.  , 0.  ]])
            >>> np.around(
            ...    ax.coords_to_point([0, 1, 1], [1, 1, 0]), 2
            ... )  # Transposed version of the above
            array([[0.  , 0.86, 0.86],
                   [0.75, 0.75, 0.  ],
                   [0.  , 0.  , 0.  ]])
```

Reply:
it gives different results on different machines 🤷🏼 i suspected it was the pixel scaling thing with virtual size but im not sure... but it gives the same result when doing frames_comparison so ... i really don't know